content/client-lib-development-guide/features.textile

```diff
@@ -869,6 +869,15 @@ enum ChannelState:
   DETACHED
   FAILED
 
+enum ChannelEvent:
+  INITIALIZED
```

Comment:
Instead of duplication, can we not simply state that it inherits / extends ChannelState?